test/cookie.js

```diff
@@ -39,6 +39,17 @@ describe('new Cookie()', function () {
       assert.throws(function () { new Cookie(function () {}) }, /argument options/)
     })
 
+    it('should accept integer maxAge options', function () {
+      var cookie = new Cookie({ maxAge: 7200000 })
+		  assert.equal(typeof cookie.maxAge, 'number')
+		  assert.equal(cookie.maxAge % 1, 0)
+	  })
+
+    it('should reject non-integer maxAge options', function() {
+	    assert.throws(function() { new Cookie({ maxAge: '7200000'}) })
```

Comment:
And these inner lines should only br 2 spaces indented from the prior level. Is it right on your side and it is just that github is messed up? Because eslint should be catching this issue if it's really there which is very confusing.